src/Components/WebAssembly/WebAssembly/src/Rendering/WebAssemblyDispatcher.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Rendering;
+
+internal sealed class WebAssemblyDispatcher : Dispatcher
```

Comment:
Can you add comments here explaining the rationale of how this context is meant to work and why? Not doc comments, just regular comments.